content/en/getting_started/synthetics/_index.md

```diff
@@ -0,0 +1,215 @@
+9. Once you've input all required fields, click the *Save Details & Record Test* button.
+
+#### Record a test with variables and assertions
+
+Tests can be only recorded from **[Google Chrome][1]**. To record your test, download the [Datadog Record Test extension for Google Chrome][2].
+
+{{< img src="synthetics/browser_tests/browser_check_record_test.png" alt="Browser test record test" responsive="true" >}}
+
+1. Optionally, select **Open in a pop-up** at the upper right of the page to open your test recording in a separate pop-up window in order to avoid sizing issues in the displayed window within Datadog's interface.
+2. Click on **Start recording** to begin recording your browser test.
+3. Your actions are recorded and used to create steps within your browser test scenario. You can record the uploading of files as an action, though this is limited to 10 files, with a limit of 5MB each.
+4. Use the actions available in the upper left corner to enrich your scenario:
+    {{< img src="synthetics/browser_tests/browser_test_step.png" alt="Browser Test steps" responsive="true" style="width:80%;">}}
+**Note**: **Your last browser test step must be an Assertion**, otherwise there is nothing to check.
+5. Click the **Variables** button under *Add New*. Select `Pattern`, `Element`, `Global Variable`, or `Email` from the dropdown menu. For example, select `Email` and Datadog will create a special email variable that allows you to test that an email was sent by the UI and received by an email server.
+6. Click the **Assertion** button under *Add New*. Select an assertion, such as `Test that an email was received`.
+    
+    [Assertions][13] allow you to check whether an element, some content, or some text is available on the current page. You can also check whether a specific email was sent. For example, you can create an assertion that checks whether an email was sent and whether specific values (string, number, regex) are present within the email subject or body. This assertion leverages email variables.
+7. Select the subject condition and click **Use variable**. Select the email variable you created. This assigns the email variable you created to the assertion. Select the body condition and click **Use variable**. Select the email variable you created.
+8. Once you have finished your Scenario, click on **Save and Launch Test**.
+
+The test overview page includes properties, history, sample results, and test results. In the top right corner, choose to **Run Test Now**, **Pause Test**, **Edit Test Details**, **Edit Recording**, **Clone**, or **Delete** a test. Under properties, add tags to keep your tests organized in the [Synthetics list][14]. All additional configuration for your tests occur in this page. To create a new test or preview existing tests, navigate back to [synthetics tests][14].
```

Comment:
Not too sure what is meant by properties? 
It might be cool to mention metrics that are being generated, screenshots on a per step basis as well as resources, errors & traces that appear when opening the context panel on one specific step (https://cl.ly/beeedaa93226)

---

At the end of browser test we focus on the browser test details page VS at the end of API test we focus on the main Synthetics homepage.
I feel like it might be useful to have a small section about Synthetics homepage at the very beginning, then detail more test details pages for both API & browser tests at the end of each respective section.